docs/guide/yaml/winrm/index.md

```diff
@@ -355,6 +355,18 @@ refer to [How and Why to re-authenticate withing a powershell script](#how-and-w
 If using the imageId of a Windows community AMI, you may find that the AMI is deleted after a few weeks.
 See [Windows AMIs on AWS](#windows-amis-on-aws) above.
 
+### VM Provisioning Times Out
```

Comment:
I'm not sure what has happened with the rebase. These changes are already in master, rather than edits in your PR. Can you try doing a `git rebase master` again please?

If they are left in, it will probably do no harm (but we should double-check after merge that it doesn't now have the text twice or anything like that)..